### ISSUE TITLE
Missing error handler in /user/getProfileImage

### ISSUE THREAD

**Issue by raydwaipayan** [SuMRoG/Our-first-project#1]
Consider the part:

```
router.get('/getProfileImage/:id', function(req, res){
  Account.find({username: req.params.id})
  .then(user=> {
    // console.log(user);
    res.json({image: user[0].image})
  })
  .catch(err=> console.log(err))
})
```

Better error handling should be there which otherwise
results in a long poll. Something on lines of:

```
catch((error)=>{
  res.status(501).message({error: error.message })
})
```

**Comment by sujal-ux** [comment]
Issue resolved

**Comment by raydwaipayan** [comment]
Fixed by 02bd590336363e73a64af4509a601f2e0dd25601
